keras2onnx/_builtin.py

```diff
@@ -978,6 +979,29 @@ def convert_tf_strided_slice(scope, operator, container):
                                   operator.inputs[0].full_name + '_identity')
 
 
+@converter_func(TYPES.Unpack)
+def convert_tf_unpack(scope, operator, container):
+    oopb = OnnxOperatorBuilder(container, scope)
+    node = operator.raw_operator
+    axis_val = node.get_attr('axis')
+    input_shape = _cal_tensor_shape(node.inputs[0])
+    if axis_val < 0 and operator.target_opset < 11:
```

Comment:
Can you explain the case where axis_val is negative? Is it the default value? Might be useful to have a comment here.

Reply:
axis_val can be negative when the user wants to specify the axis counting back from the last dimension. Negative axis is commonly used.